packages/sdk-codegen-scripts/scripts/utils.ts

```diff
@@ -42,10 +42,10 @@ const homeToRoost = '../../../'
 const getRootPath = () => path.join(__dirname, homeToRoost)
 const rootFile = (fileName = '') => path.join(getRootPath(), fileName)
 
-export const apixSpecFileName = (fileName: string) => {
-  const p = path.parse(fileName)
-  return rootFile(`spec/${p.base}`)
-}
+// export const apixSpecFileName = (fileName: string) => {
+//   const p = path.parse(fileName)
+//   return rootFile(`spec/${p.base}`)
+// }
```

Comment:
maybe remove if unused (unless future plans then maybe leave a comment about it)?

Reply:
trivial code. No reason to keep